lang/cs/Org.Apache.REEF.IMRU.Tests/TestActiveContextManager.cs

```diff
@@ -181,10 +182,10 @@ private static IFailedEvaluator CreateMockFailedEvaluator(IList<int> ids)
         /// <summary>
         /// A Context Manager observer for test
         /// </summary>
-        private sealed class TestContextObserver : IObserver<IDictionary<string, IActiveContext>>
+        private sealed class TestContextObserver : IObserver<IEnumerable<IActiveContext>>
```

Comment:
We changed interface to use IEnumerable<IActiveContext>. So the test needs to follow the same signature.